content/api/commands/clearallcookies.md

```diff
@@ -0,0 +1,75 @@
+---
+title: clearAllCookies
+---
+
+Clear all browser cookies.
+
+<Alert type="warning">
+
+Cypress automatically clears all cookies _before_ each test to prevent state
+from being shared across tests. You shouldn't need to use this command unless
+you're using it to clear a specific cookie inside a single test.
```

Comment:
```suggestion
you're using it to clear a specific cookie inside a single test or test isolation is disabled.
```

Reply:
Addressed in https://github.com/cypress-io/cypress-documentation/pull/4918/commits/457c00ea98fdf8f01dc90a523f8cb85b7eb3c7d9. I aligned all these warnings, here in the type definitions.